app/controllers/meetings/google_calendar_event_controller.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+module Meetings
+  class GoogleCalendarEventController < ApplicationController
+    before_action :set_meeting
+    before_action :set_meeting_member
+
+    # POST /meetings/:meeting_id/google_calendar_event
+    def create
+      message = create_event
+
+      redirect_to(group_path(@meeting.group_id), notice: message)
+    end
+
+    # DELETE /meetings/:meeting_id/google_calendar_event
+    def destroy
+      message = destroy_event
+
+      redirect_to(group_path(@meeting.group_id), notice: message)
```

Comment:
What happens when the message is an error? I think we should be using alert instead of notice.